### PR TITLE
github(issue_template): add some info

### PR DESCRIPTION
**Pull request by SukkaW**
## What does it do?

* Add special notice for zh_CN users (because most of them just won't follow issue template)
* Add a step about Safe mode debugging
* Provide a tip for using code blocks

## How to test

No test is needed for this PR.

## Screenshots

https://github.com/SukkaW/hexo/blob/1ecce8e6de96a49bdf891d252dda2d5d6d9ff9f6/.github/ISSUE_TEMPLATE/bug_report.md

## Pull request tasks

- [ ] Add test cases for the changes.
- [x] Passed the CI test.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -35,16 +39,33 @@ Please check followings before submitting a new issue.
...
+## Is the problem still there under "Safe mode"?
+
+<!--
+https://hexo.io/docs/commands#Safe-mode
+
+"Safe mode" will disable all the plugins and scripts.
+If your problem disappear under "Safe mode" means the problem is probably at your newly installed plugins, not at hexo.
+-->
+
 ## Environment & Settings
```
Comment (YoshinoriN) [inline .github/ISSUE_TEMPLATE/bug_report.md]: As you know. A while ago one of the hexo user submit his/her secret server information. 
I want to add a message just like `PLEASE DO NOT SUBMIT SECRET INFORMATION`.

Comment (SukkaW) [reply .github/ISSUE_TEMPLATE/bug_report.md]: `Please DO NOT CONTAINS SECRET INFORMATION (likes password or GitHub Personal Access Token) when submit a new issue`.

Is that ok?

Comment (SukkaW) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Added.

**Comment by YoshinoriN** [comment]
@curbengh 
Would you please check the Chinese?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -9,10 +9,16 @@ assignees: ''
 
 <!-- NOTE:
 
+Those information will be helpful for us to investigate the problem. So please, follow the steps we provided exactly when submitting your issue, and do not remove the issue template.
```
Comment (curbengh) [inline .github/ISSUE_TEMPLATE/bug_report.md]: "A good bug report should also have your configuration and build environment information, which are essential for us to investigate the problem. We've provided the following steps on how to attach the necessary information."

Sounds a bit more neutral.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -9,10 +9,16 @@ assignees: ''
...
+
 If you find that markdown files are not rendered as expected, please go to https://marked.js.org/demo/ to see if it can be reproduced there. If it can be reproduced, please file a bug to https://github.com/markedjs/marked.
 
 If you want help on your bug, please also send us the git repository (GitHub, GitLab, Bitbucket etc...) where your hexo code is stored. It would greatly help. If you prefer not to have your hexo code out in public, please upload to a private GitHub repository and grant read-only access to hexojs/core.
 
+Please DO NOT CONTAIN SECRET INFORMATION (likes password or GitHub Personal Access Token) when submitting a new issue.
```
Comment (curbengh) [inline .github/ISSUE_TEMPLATE/bug_report.md]: "Please take extra precaution not to attach any secret environment variables (e.g. password and token)."

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -9,10 +9,16 @@ assignees: ''
...
+
+针对简体中文用户的提示：请在提交 issue 时完全按照我们提供的步骤进行操作，不要删除问题模板。按照步骤提供相关信息将有助于我们调查问题。Hexo 维护团队并非全部是简体中文使用者，使用英语描述你的问题可以让更多的人帮助到你。下述步骤使用英语写就，必要时请使用翻译网站或软件。
```
Comment (curbengh) [inline .github/ISSUE_TEMPLATE/bug_report.md]: It's a bit inappropriate to target zh-CN users only :sweat_smile: 
The suggestion I provided sounds more positive and encouraging.

In regards to "Hexo 维护团队并非全...", I would say,

"请尽量使用英语描述你的问题，这可以让更多的人帮助到你。"

I don't think we need to mention translation website/software, users would use if they feel necessary.

Comment (SukkaW) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Updated.

**Comment by SukkaW** [comment]
@curbengh Yet another issue (#3988) is in Chinese while delete issue template at all. Are you still believe it is inappropriate to target zh-CN users only?

I'm even considering add an issue template specifically for zh-CN users.

**Comment by curbengh** [comment]
> Are you still believe it is inappropriate to add a tip target zh-CN users only?

Mandarin text itself is already target them, so "针对简体中文用户" is unnecessary.

suggestion:

"须按照步骤提供相关信息，这将有助于我们调查你的问题。请尽量使用英语描述你的问题，这可以让更多的人帮助到你。"

the "must" at the start should already send a strong message. if users still don't follow it, they would also ignore "在提交 issue 时请不要删除下面的模板" too.

> I'm even considering adding an issue template specifically for zh-CN users.

+1 if github can do this

**Comment by SukkaW** [comment]
I believe it is much more important to add zh-CN specific instruction in issue template. There are more and more Chinese user just simply delete issue template and doesn't provide any useful information.

I am going to force this PR to merge then.

#4075 #4148 #4159

**Comment by SukkaW** [comment]
@stevenjoezhang Done.

**Pull request merged**